### PR TITLE
fix: Set missing test environment variable

### PR DESCRIPTION
**Pull request by bemwen**
## Description

Sets the missing environment variable in domain tests

## References

* #879

### CODE REVIEW

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Fixtures/ChargesFunctionAppFixture.cs [hunk 1]
```diff
@@ -168,7 +168,7 @@ await ServiceBusResourceProvider
 
             var chargePricesUpdatedTopic = await ServiceBusResourceProvider
                 .BuildTopic(ChargesServiceBusResourceNames.ChargePricesUpdatedTopicKey).SetEnvironmentVariableToTopicName(EnvironmentSettingNames.ChargePricesUpdatedTopicName)
-                .AddSubscription(ChargesServiceBusResourceNames.ChargePricesUpdatedSubscriptionName)
+                .AddSubscription(ChargesServiceBusResourceNames.ChargePricesUpdatedSubscriptionName).SetEnvironmentVariableToSubscriptionName(EnvironmentSettingNames.DefaultChargeLinksDataAvailableNotifiedSubscription)
```
Comment (BjarkeMeier) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Fixtures/ChargesFunctionAppFixture.cs:171]: Are you sure this is the correct env setting name?

Comment (prtandrup) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Fixtures/ChargesFunctionAppFixture.cs:171]: Not sure I understand why `EnvironmentSettingNames.DefaultChargeLinksDataAvailableNotifiedSubscription` is being set with `ChargePricesUpdatedSubscriptionName`?

Comment (bemwen) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Fixtures/ChargesFunctionAppFixture.cs]: Right setting name, wrong value. Fixed

Comment (bemwen) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Fixtures/ChargesFunctionAppFixture.cs]: Fixed, got to late last night appearently

**Review by prtandrup — APPROVED** [review]

**Pull request merged**